### PR TITLE
UGDG - Include Personalised Messages 

### LINKED ISSUE #174: [PE-D] Mismatch in expected result in userguide and actual result
In the userguide, this was the expected result when adding a task.

However when testing , the following image shows the actual result.

I think it would be good to update the user guide to show the actual result?

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: xingrong123/ped#1

### LINKED ISSUE #199: DG - Include personalised messages into both UGDG

### PR DESCRIPTION
**Pull request by yAOwzers**

### CODE REVIEW

**Comment by yAOwzers** [comment]
Adjusted DG based off Rishi's comments as well. PLEASE TELL ME IF WE ARE GnA PUT THE PERSONALISED MSG IN 3.3

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -284,15 +284,17 @@ The `Page` object,
...
-* Contains the method `saveToFile` to save the current AppState of the application in the `notebooks.txt` and `tasks.txt` files.  
+- Stores user's inputs into .txt files, which are all found under the same `data` directory.  
 
-* Contains the method `readFromFile` to load up data containing the input of the user's previous session on Zer0Note.  
+- Contains the method `saveToFile` to save the current AppState of the application in the `notebooks.txt` and `tasks.txt` files.  
 
-* Saves the name of the user in a `nameOfUser.txt` and reads it back in the form of personalised messages.   
+- Contains the method `readFromFile` to load up data containing the input of the user's previous session on Zer0Note.  
+
+- Saves the name of the user in a `nameOfUser.txt` and reads it back in the form of personalised messages.
```
Comment (franceneee) [inline docs/DeveloperGuide.md:298]: Nice! more specific now. :)

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -402,13 +404,15 @@ The following sequence diagram shows how the mark as done operation works:
...
 
-Step 3. The `execute()` method proceeds to parse the user's intended task index to be marked as done through `Integer.parseInt()`. Using this index, it initialises a varible `taskDone` of type task and calls `AppState#markAsDone(index)`.  
+3. The `execute()` method proceeds to parse the user's intended task index to be marked as done through `Integer.parseInt()`. Using this index, it initialises a variable `taskDone` of type task and calls `AppState#markAsDone(index)`.  
+        
+4. `Appstate#markAsDone(index)` calls the `isDone()` method in task, which changes the isDone variable in the specific task to be true.  
 
-Step 4. After `taskDone` is initialised, a `messages` of type CliMessages calls a method `printMarkDone(taskDone)` with the variable `taskDone` as the argument, which in turn prints a success message with the respective task to the user.   
+5. After `taskDone` is initialised, a `messages` of type CliMessages calls a method `printMarkDone(taskDone)` with the variable `taskDone` as the argument, which in turn prints a success message with the respective task to the user.
```
Comment (franceneee) [inline docs/DeveloperGuide.md]: wait, is `messages` an object of type CliMessages? I think the English is a bit confusing here. xD

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -284,15 +284,17 @@ The `Page` object,
 
 ### 3.6. Storage Component
 
-![UML diagram for Storage](diagrams/class/jpeg/Storage_UML_class.jpeg)
+![UML diagram for Storage](diagrams/class/jpeg/Storage_UML_class.jpg)
```
Comment (franceneee) [inline docs/DeveloperGuide.md:288]: wait so did you fix the image?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -92,14 +93,33 @@ Here are some icons that will be used in this User Guide:
...
-    ```
-    >>> select /sChapter 1
-        now in notebook section: Chapter 1
-    ```
+* Within a code segment, the symbols before the `$` symbol indicates the current mode and notebook/section the user is in.
+    * The following symbols represents the respective address:
+        * `T:$` - The user is in TimeTable mode.  
+        * `N:$` - The user is in Notebook mode.
+        * `N/[NOTEBOOK NAME]` - The user is in Notebook mode, and is in the indicated notebook with 'NOTEBOOK NAME'.
+        * `N/[NOTEBOOK NAME]/[SECTION NAME]` - The user is in Notebook mode, and is in the indicated section with 'SECTION NAME' of the notebook with 'NOTEBOOK NAME'.
```
Comment (franceneee) [inline docs/UserGuide.md]: It displays even up to N/[NOTEBOOK NAME]/[SECTION NAME]/[PAGE NAME] actually...

File: docs/UserGuide.md [hunk 5]
```diff
@@ -168,7 +188,40 @@ Format: `mode /t`
...
+T:$ add /tRead book /by19-10-2020 1800
+-------------------------------------------------------------
+Added: Read Book
+1:[x] Read book (by: Oct 19 2020 06.00 PM)
+-------------------------------------------------------------
+{personalised message}
+-------------------------------------------------------------
+```
+
+> :exclamation: {personalised message} is a placeholder for the various personalised message that a user will receive.
```
Comment (franceneee) [inline docs/UserGuide.md:222]: YAYYYYY NICE!

File: docs/UserGuide.md [hunk 6]
```diff
@@ -168,7 +188,40 @@ Format: `mode /t`
...
+
+```
+T:$ add /tRead book /by19-10-2020 1800
+-------------------------------------------------------------
+Added: Read Book
+1:[x] Read book (by: Oct 19 2020 06.00 PM)
+-------------------------------------------------------------
+{personalised message}
+-------------------------------------------------------------
+```
```
Comment (franceneee) [inline docs/UserGuide.md:220]: Nice :P

**Review by franceneee — COMMENTED** [review]
Omg, thank you so much for your effort. THANK YOU FOR YOUR HARD WORK JOEL.

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (yAOwzers) [reply docs/DeveloperGuide.md]: Oh yes, `message` is an object. Will it be clearer if included a  `CliMessage` object name `messages` instead?

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (yAOwzers) [reply docs/DeveloperGuide.md:288]: yeahhhhh

File: docs/UserGuide.md (hunk 4 above)
Comment (yAOwzers) [reply docs/UserGuide.md]: OH LOL
what the hek HAHHAHA ok i go edit

**Review by franceneee — APPROVED** [review]

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (franceneee) [reply docs/DeveloperGuide.md]: After `taskDone` is initialised, `messages` an object of type CliMessages calls a method `printMarkDone(taskDone)` with the variable `taskDone` as the argument, which in turn prints a success message with the respective task to the user. ** ?

**Comment by yAOwzers** [comment]
@chuckiex3 your comments ah... really phenomenal! I've made the changes!

**Review by Lusi711 — APPROVED** [review]

**Pull request merged**